corehq/apps/reports_core/filters.py

```diff
@@ -197,7 +197,7 @@ def __init__(self, name, required=True, label='Choice List Filter',
         self.choices = choices or []
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
+        choice = unicode(kwargs[self.name])
```

Comment:
I think fine, unless you're worried that `kwargs[self.name]` may be `None` or some other value that you don't want to return the unicode version of.